sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsPartitionManagement.java

```diff
@@ -80,9 +79,14 @@ void createPartition(
      * @return true if the partition exists, false otherwise
      */
     default boolean partitionExists(InternalRow ident) {
```

Comment:
can we also update the API doc of this method?

Reply:
What do you propose? The doc says clearly that the method tests existence of **A** partition. Do you want to say that `ident` must contain values for ALL partition fields?

Reply:
how about `@param ident a partition identifier which must contain all partition fields in order`?